app/controllers/analytics_controller.rb

```diff
@@ -67,7 +67,7 @@ def epi_stats
     # Get analytics from most recent cache analytics job
     most_recent_analytics = current_user.jurisdiction.analytics.last
 
-    return nil if most_recent_analytics.nil?
+    return {} if most_recent_analytics.nil?
```

Comment:
Good point, so the reason why it was originally written this way as `return nil` is because I wanted to handle this logic in the front end and it logically makes more sense to return `nil` instead of an empty object. The reason for this change is actually because this was causing a bug where if a user tries to visit the analytics page when there are no analytics yet, rails will try to redirect the user to the home page.

In the case of an epi trying to view it, it'll just redirect to the public health page (I was pretty confused why this was happening at first and thought the button might be broken or something, but this was the reason why)

In the case of an analyst trying to view it, it'll try to keep redirecting over and over again and crash.

I've updated the front end accordingly to handle this logic in this line here: https://github.com/SaraAlert/SaraAlert/blob/SARAALERT-483/app/javascript/components/analytics/MonitorAnalytics.js#L23

Reply:
Ah okay, thanks for the explanation!

Reply:
no problem!